player.cpp

```diff
@@ -650,6 +650,47 @@ void Player::makeMove() {
       else
         getView()->presentText("Sorry", "Couldn't parse \"" + action.get<string>() + "\"");
       break;
+    case UserInputId::CHANGE_TRIBE: {
+      //Michal, is there a better way to do this?
+	  string tribe = action.get<string>();
+      if (tribe== "Monster")
```

Comment:
The common style in KeeperRL is for operators like == to have a space on both sides.